vendor/engines/secure_rooms/app/models/secure_rooms/order_detail_extension.rb

```diff
@@ -6,6 +6,9 @@ module OrderDetailExtension
 
     included do
       has_one :occupancy, dependent: :destroy, inverse_of: :order_detail, class_name: SecureRooms::Occupancy
+
+      scope :non_occupancies, -> { joins(:product).where("products.type <> 'SecureRoom'") }
```

Comment:
Are we using this scope anywhere? I actually got rid of the analogous `non_instruments` scope in #1013.